catalog/ui/catalog-ui-search/src/main/webapp/properties.js

```diff
@@ -81,13 +81,21 @@ define(function (require) {
                 throw new Error('Configuration could not be loaded: (status: ' + status + ', message: ' + errorThrown.message + ')');
             });
 
+            this.handleEditing();
+
             return props;
         },
+        handleEditing: function(){
+            $('html').toggleClass('is-editing-restricted', this.isEditingRestricted());
```

Comment:
@codice/security We need to talk about how we plan to enforce options like this on the backend as well.